content/en/project/status/2023MedleyAnnualReport.md

```diff
@@ -2,6 +2,8 @@
 title: 2023 Annual Report
 weight: 1
 type: docs
+aliases:
+ - /news/2022medleyannualreport/
```

Comment:
Shouldn't this link to `/project/status/2022medleyannualreport/`? If the user is searching for 2022 we should take them there and not 2023.

Reply:
moved link